.env

```diff
@@ -0,0 +1,7 @@
+PORT=3000
```

Comment:
Usually these are not committed and it is the same as the defaults you have in the code. Thoughts?

Reply:
I was looking up the docs for the `.env` file and they did recommend committing it. And if you want to set different settings create a `env.local` which overrides the `.env` file. 

https://create-react-app.dev/docs/adding-custom-environment-variables/

As for it already being the same as the default. I added it to the file so it's more clear that it can be changed. If you think it makes more sense to remove it though I'll do that.